Updated marketplace cards

electrovir opened the pull request:
- Created a `DropShadowCard` component for applying the design system shadow style.
- Created a `NftCard` component (which will be re-used for several views) which uses `DropShadowCard`.
- Use the new `NftCard` component for the collection cards in the Marketplace view:
    ![new-marketplace](https://user-images.githubusercontent.com/1205860/181667370-043de2ce-b103-4b03-b5b8-d4c396b5afd4.png)
    - On small screens the cards shrink in width and wrap:
        ![skinny-wrapping](https://user-images.githubusercontent.com/1205860/181667681-6e846bf2-c68b-430f-a5e4-c070e6e71d99.png)
    - If the brief is too long on a small screen it will overflow with ellipsis:
        ![overflowing-brief](https://user-images.githubusercontent.com/1205860/181666255-b72ce847-8ec2-400a-9d2a-2fa6d885b484.png)
    - It a collection has a long name, it will be taller than the default card height:
        ![tall-collection-card](https://user-images.githubusercontent.com/1205860/181667547-9fddbca9-96d9-4606-8463-9048ff91c422.png)

bodily11 commented:
What is the max length of a title? What if someone tries to be funny and create a title that is 100 words long? Do we ellipses on the title after the second line of the title? Growing forever is probably not ideal, haha.

electrovir commented:
Rebased and added collection name limiting to 2 lines. It looks like this:

<img src="https://user-images.githubusercontent.com/1205860/183552659-e663b2ee-4f27-45b5-b4a3-c1d02cdcdcae.png" width="300px" />
